utils.py

```diff
@@ -159,3 +159,13 @@ def _compare_endpoints(endpoint1, endpoint2):
     elif 'out_vlan' in endpoint1 or 'in_vlan' in endpoint2:
         return False
     return True
+
+
+def match_field_dl_vlan(vlan_value, field_flow):
+    """Verify match in dl_vlan"""
+    special = {"any": "4096/4096", "untagged": 0}
+    vlan_value = special.get(vlan_value, vlan_value)
+    if isinstance(vlan_value, int):
+        return vlan_value == field_flow
+    value, mask = map(int, vlan_value.split('/'))
+    return value & (mask & 4095) == field_flow & (mask & 4095)
```

Comment:
@Alopalao, Thanks for pointing out these cases.
Note that in the latest version that I have uploaded, the possible options (int, special cases, str with `/`) are taken into account for both `vlan_value` and `field_flow`.

Reply:
Aldo, if you see another corner case, let Gretel knows please. The latest version that Gretel pushed is differentiating when it's a int or str, I have found other issues that I opened new threads, but this one here looks good to me. Otherwise we can close this thread. Thank you both.